pkg/policy/custompolicybuilder/custom-policy-builder.go

```diff
@@ -0,0 +1,109 @@
+package custompolicybuilder
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/soluble-ai/soluble-cli/pkg/policy"
+	"gopkg.in/yaml.v3"
+)
+
+var inputTypeForTarget = map[policy.Target]string{
```

Comment:
This is already defined in `pkg/policy/opal/opal.go` rather than have a repeated code block you could extract this into the policy.go and refactor this and opal.go to use it